netty/build.gradle

```diff
@@ -17,7 +17,8 @@ evaluationDependsOn(project(':grpc-core').path)
 
 dependencies {
     api project(':grpc-core'),
-            libraries.netty
+            libraries.netty,
+            libraries.netty_epoll
```

Comment:
This change seems wrong. Epoll shouldn't be part of the API surface. Why was it made?

Reply:
Things like `io.netty.channel.epoll.Epoll` , `io.netty.channel.unix.DomainSocketAddress` that are used inside `UdsNameResolverProvider.java` needed `netty_epoll` library.

Reply:
Still unresolved:

> Epoll JNI also might not be available at runtime, so the ChannelProvider will need to call Epoll.isAvialable() from isAvailable() (probably indirectly via UDS_CHANNELS_EVENT_LOOP_GROUP != null).